Sources/Apodini/Semantic Model Builder/SemanticModelBuilder.swift

```diff
@@ -89,7 +92,7 @@ class SemanticModelBuilder: InterfaceExporterVisitor {
                     return AnyHandlerIdentifier(handlerIndexPath.rawValue)
                 }
             }(),
-            handler: handler,
+            handler: handler.inject(app: app),
```

Comment:
I would move this to the creation of the `guards` and `responseTransformers`, it seems a bit hidden in the initializer here and I almost missed it.